data/neo4j/neo4j_database.py

```diff
@@ -0,0 +1,60 @@
+from neo4j import GraphDatabase
+from typing import Optional
+
+query_periodic_commit = "USING PERIODIC COMMIT 10000 "
+
+
+class Database:
+    def __init__(self, uri: str, user: str, password: str):
+        self.driver = GraphDatabase.driver(uri, auth=(user, password))
+
+    def close(self):
+        self.driver.close()
+
+    """ GET """
+
+    @staticmethod
+    def get_node(node_id: int, node_type: str, no_return: bool = False) -> str:
+        query = f"MATCH(n:{node_type} {{_id:'{str(node_id)}'}})"
+
+        if no_return:
+            return query + " "
+        else:
+            return query + " RETURN n"
+
+    @staticmethod
+    def get_n_neighbors(
```

Comment:
I don't think we need staticmethods, these can be private, pure functions, outside of the scope, if they're not exported. that's a great convention! staticmethods are only useful if we want to export pure functions, otherwise we just get confused whether these are private or public functions.